apps/workers/sql-readx/worker.go

```diff
@@ -26,22 +26,61 @@ type worker struct {
 	Query  string
 }
 
+type executer struct {
+	task.Meta
+	db    *sqlx.DB
+	Query string
+}
+
 type FieldMap map[string]string
 
 func (o *options) NewWorker(info string) task.Worker {
 	// unmarshal info string
 	iOpts := struct {
+		Exec        bool              `uri:"exec"`
 		Table       string            `uri:"table"`
 		QueryFile   string            `uri:"origin"` // path to query file
+		Query       string            `uri:"query"`
```

Comment:
does the name query make sense? would calling it statement be better?

Reply:
maybe command? statement is fine though too

Reply:
I think query is fine.